.github/workflows/workflow-vercel-deployment.md

```diff
@@ -0,0 +1,15 @@
+## Vercel Preview Deployment
+
+   - **Purpose:** Deploy project artifacts to a Vercel preview environment and comment the deployment URL on the pull request for review.
+   - **Usage:** Trigger this workflow using `workflow_call`. You need to set `VERCEL_ORG_ID`, `VERCEL_PROJECT_ID` as secrets in your GitHub repository.
+   - **Required Secrets:**
+     - `VERCEL_TOKEN`: Token for authentication with Vercel, enables interaction with your Vercel account.
+     - `GITHUB_TOKEN`: Generated by GitHub to enable actions inside the repository, like commenting on pull requests.
+   - **Workflow Steps:**
+     - **Checkout Repository**: Gets code from the repository.
```

Comment:
this seems excessive and redundant doc, listing every step. requires maintaining this doc everytime we add, remove or modify a step. doc should not repeat what can be read from the code.

Reply:
done